orchestrators/ProcurementOperations/src/DTOs/VendorRecommendation/VendorRecommendationScoredCandidate.php

```
@@ -0,0 +1,65 @@
+    /**
+     * @param list<string> $deterministicReasons
+     * @param list<string> $llmInsights
+     * @param list<string> $warningFlags
+     * @param list<string> $warnings
+     */
+    public function __construct(
+        public string $vendorId,
+        public string $vendorName,
+        public int $fitScore,
+        public string $confidenceBand,
+        public string $recommendedReasonSummary,
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $recommendedReasonSummary. Keep variable name length under 20. (undefined)

(LongVariable)